packages/design-system/package.json

```diff
@@ -12,7 +12,7 @@
     "start": "npm run storybook",
     "test": "talend-scripts test",
     "test:cy": "cypress run-ct",
-    "test:demo": "build-storybook --docs && node ./scripts/sitemap.js",
+    "test:demo": "npm run build-storybook",
     "storybook": "npm run build:lib && BROWSER=none start-storybook -p 6006",
     "build-storybook": "npm run build:lib && build-storybook --docs && node ./scripts/sitemap.js",
```

Comment:
```
"test:demo": "build-storybook --docs && node ./scripts/sitemap.js",
"build-storybook": "yarn build:lib && yarn test:demo",
```

we don't want to run the build when we call `test:demo` because everything is already built